scripts/export-users.py

```diff
@@ -0,0 +1,372 @@
+#!/usr/bin/env python3
+
+"""
+User export script to aid in migration
+
+This script gets users and their assigned cohorts from Fusion Auth to Armadillo.
+It will group each user by cohort and will export it as TSV to a given export location, so that upon
+request this TSV can be used to set users to their appropriate cohorts.
```

Comment:
Isn't this "just exporting Fusion users into a TSV? Or is it a special Armadillo compatible format?

Reply:
Simple TSV, I'll change it